### DISCUSSION TITLE
1.22

### DISCUSSION THREAD

**Discussion by PalmerAL** [minbrowser/min#1774] [General]
New in this release:

* The searchbar can now perform math calculations (by @lubiedo). Try "sqrt(2) + 1".
* The task overlay now has a search field to search through your open tabs.
* Min now follows your operating system's dark mode preference by default (by @rost).
  * If you want to switch back to the old behavior of using a dark theme at night, you can do so from preferences.
* Page translation (available by right-clicking on a page) now supports more languages.
* Added a keyboard shortcut to rearrange tabs (by @rost).
* Upgraded to Chromium 94 / Electron 15.
* Translation updates.

Bug fixes:
* "Hide others" now uses the correct keyboard shortcut on macOS.
* Fixed an issue where pressing cmd+s with a file open in a tab wouldn't save the file correctly.
* Fixed an issue where popups sometimes wouldn't show a URL in the URL bar.
* Fixed an issue where the download manager would show the wrong file name if you renamed a file (by @lubiedo)

<hr /><em>This discussion was created from the release <a href='https://github.com/minbrowser/min/releases/tag/v1.22.0'>1.22</a>.</em>

**Comment by tvardy** [comment]
darwin-arm64 build says "the app is damaged and cannot be open" :(

**Reply by PalmerAL** [reply]
Can you try the steps listed here? https://github.com/minbrowser/min/issues/1633

**Reply by tvardy** [reply]
Yes! Thanks! [this](https://github.com/minbrowser/min/issues/1633#issuecomment-855364582) post helped
After that, the `Min.app` can be moved to `/Applications` folder and still works fine.
